cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -65,6 +66,11 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run existing copyrigh
         ).file(mustExist = true, canBeFile = true, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
     ).single().required()
 
+    private val skipExcluded by option(
+        "--skip-excluded",
+        help = "Do not scan excluded projects or packages. Works only with the '--ort-file' parameter."
+    ).flag(default = false)
```

Comment:
I'd prefer to omit the default value here as we do in other places for `flag()`, as it's fair to assume that just declaring a flag should not enable it.

Reply:
Done.